src/image_manager/test/test_SkopeoDriver.cpp

```diff
@@ -242,7 +242,9 @@ TEST(SkopeoDriverTestGroup, acquireAuthFile) {
             CHECK_FALSE(boost::filesystem::exists(expectedAuthFilePath));
```

Comment:
With the auth file moving to a random path, it makes little sense to check whether a hardcoded path exists or not. I agree this is unnecessary and can be removed.